cinema/models.py

```diff
@@ -111,13 +121,15 @@ def save(
     ):
         self.full_clean()
         super(Ticket, self).save(
-            force_insert, force_update, using, update_fields
+            force_insert,
+            force_update,
+            using,
+            update_fields
         )
 
     def __str__(self):
-        return (
-            f"{str(self.movie_session)} (row: {self.row}, seat: {self.seat})"
-        )
+        return f"{str(self.movie_session)} " \
```

Comment:
why did you do it? it is better code 

return (
            f"{str(self.movie_session)} (row: {self.row}, seat: {self.seat})"
        )